lib/base16.rb

```diff
@@ -0,0 +1,10 @@
+# See https://www.rfc-editor.org/rfc/rfc4648#section-8
+class Base16
```

Comment:
😂 this is simple but we should see if we can share this somehow

Maybe we fork https://github.com/sspreitzer/base16 or just make our own?